pkgs/stdenv/darwin/default.nix

```diff
@@ -402,6 +402,10 @@ rec {
             inherit (selfDarwin) sigtool;
           };
 
+          postLinkSignHook = darwin.postLinkSignHook.override {
+            inherit (selfDarwin) sigtool;
+          };
+
```

Comment:
We pass down `postLinkSignHook` from `stage0`. Without this we get:

```
output '/nix/store/6c0d8s1q70dbr73d7rjvlkvi1hlv3bnh-bootstrap-stage2-stdenv-darwin' is not allowed to refer to the following paths:
  /nix/store/0a7jd4wv0jicfn9czi136rir8wp6xnhm-cctools-port-949.0.1
  /nix/store/qwjr0yyapjjfcbw2mqp0ni97dbccjv12-ncurses-6.2
  /nix/store/xw2gjbxp31c01flxgnahsid2b0iva9sa-libtapi-1100.0.11
```

If we're not allowed to use `postLinkSignHook` with the real `cctools-ports` in `stage2`, then I don't see what else we're supposed to use in that stage.